src/mlpack/tests/nbc_test.cpp

```diff
@@ -60,16 +61,27 @@ BOOST_AUTO_TEST_CASE(NaiveBayesClassifierTest)
 
   arma::mat testData;
   arma::Mat<size_t> testRes;
+  arma::mat testResProba;
   arma::Row<size_t> calcVec;
+  arma::mat calcProba;
   data::Load(testFilename, testData, true);
   data::Load(testResultFilename, testRes, true);
+  data::Load(testResultProbaFilename, testResProba, true);
```

Comment:
I used GaussianNB module in scikit-learn to calculate posteriors. I don't know if it's the appropriate way to do such testing.